server/server/src/account_service.rs

```diff
@@ -147,22 +152,54 @@ pub enum GetUsageHelperError {
     UserNotFound,
 }
 
-pub fn get_usage_helper(
+pub fn get_usage_helper<T>(
+    tree: &mut LazyTree<T>, sizes: &HashMap<Uuid, u64>,
+) -> Result<Vec<FileUsage>, ServerError<GetUsageHelperError>>
+where
+    T: TreeLike,
+{
+    let ids = tree.owned_ids();
+    let root = ids.iter().find(|file_id| match tree.find(file_id) {
+        Ok(f) => f.is_root(),
+        Err(_) => false,
+    });
+
+    if root.is_none() {
```

Comment:
you can either `ok_or` or `let-else` to get rid of the optional status. Then you won't be unwrapping below. From a previous comment: 
<img width="809" alt="image" src="https://github.com/lockbook/lockbook/assets/821972/6ae855e5-f10e-4d10-a168-e6b6b2309407">